src/django_wplogin/__init__.py

```diff
@@ -2,4 +2,4 @@
 Enhance the Django admin login page
 """
 
-__version__ = "3.0.1"
+__version__ = "3.0.2"
```

Comment:
I don't like the number 2. Make it `3.0.4`

Reply:
Makes sense